gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1515,6 +1516,57 @@ def test_{{ method_name }}_rest_unset_required_fields():
     {% endif %}{# required_fields #}
 
 
+{% if not (method.server_streaming or method.client_streaming) %}
```

Comment:
just in case, do bidirectional streaming methods have their own flag or are signified by both server_streaming and client_streaming being true?